graviti/manager/draft.py

```diff
@@ -57,6 +59,23 @@ def __init__(
         self.created_at = created_at
         self.updated_at = updated_at
         self.description = description
+        self.operations: Set[Union[SheetPost, SheetGet, SheetDelete]] = set()
+
+    def __getitem__(self, key: str) -> DataFrame:
+        value = self._get_data().__getitem__(key)
+        if SheetPost(key) not in self.operations:
+            self.operations.add(SheetGet(key))
```

Comment:
why we need to add operations for `__getitem__`, `__getitem__` should be a read-only move?

Reply:
We need to know which sheets may have been modified. Then we don't need to go through all the sheets.